final_task/rss_reader/article.py

```diff
@@ -0,0 +1,75 @@
+from string_operations import *
+import logging
+
+
+class Article:
+    """single article class"""
+
+    def __init__(self, parsed):
+        """receive parsed article and extracts data from it"""
+        logging.info('      title')
+        self.title = make_string_readable(parsed.title)
+
+        logging.info('      link')
+        self.link = parsed.link
+
+        logging.info('      date')
+        self.published = extract_date(parsed)
+
+        logging.info('      summary')
+        summary_ = extract_topic_info_from_summary(parsed.summary)
+        self.summary = make_string_readable(summary_)
+
+        logging.info('      image links')
+        self.media = parsed.media_content
+
+        logging.info('      image description')
+        description_ = extract_image_info_from_summary(parsed.summary)
+        self.media_description = make_string_readable(description_)
```

Comment:
Here and below: there is no need to make logging that detailed. Just logging the start and end of the function will suffice.

Reply:
Fixed